src/sentry/models/event.py

```diff
@@ -535,6 +512,26 @@ def id(self):
         # have to reference the row id anyway.
         return self.event_id
 
+    @property
+    def timestamp(self):
+        return self.snuba_data["timestamp"]
+
+    @property
+    def event_id(self):
+        return self.snuba_data["event_id"]
+
+    @property
+    def project_id(self):
+        return self.snuba_data["project_id"]
+
+    @property
+    def group_id(self):
+        return self.snuba_data["group_id"]
+
+    @group_id.setter
+    def group_id(self, value):
+        self.snuba_data["group_id"] = value
```

Comment:
who is doing this ?

Reply:
EventCommon does this https://github.com/getsentry/sentry/blob/master/src/sentry/models/event.py#L95

We can likely get rid of it once we deprecate Event, but I didn't want to change EventCommon for now.

Reply:
I just realised we need the same for project_id